config/misc/iatlas-runner.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: runner.yaml
```

Comment:
To be more secure i think it would be better to separate the runners.  one runner for staging and one runner for prod.

Reply:
I've separated the runner and kms resources in a recent PR update... there are two sets of these resources: one for prod and one for staging. It wasn't clear how to me how to use sceptre-identity-resolver to do this, so I just used an Environment parameter to associate a runner with a db with a kms key.